plan/task_profile.go

```diff
@@ -63,8 +66,7 @@ func (t *copTaskProfile) copy() taskProfile {
 }
 
 func attachPlan2Task(p PhysicalPlan, t taskProfile) taskProfile {
```

Comment:
s/attachPan2Task/attachPlan2TaskProfile

To keep it consistent with the method in Plan.